include/grpc++/security/server_credentials.h

```diff
@@ -39,6 +39,7 @@
 
 #include <grpc++/security/auth_metadata_processor.h>
 #include <grpc++/support/config.h>
+#include <grpc/grpc_security.h>
```

Comment:
@ctiller What are you thoughts on this? Is there a better way I can include the enum in c++ without adding a dependency to the c code directly.